### DISCUSSION TITLE
some text color style do not take effect until a few seconds after the page loaded

### DISCUSSION THREAD

**Discussion by kekeqy** [vikejs/vike#565] [Discussion]
### Description

我发现一个奇怪的问题，有些文本的颜色样式要在页面加载几秒后才生效，有很大概率会发生这种情况。我检查了样式的加载顺序，首先加载的是子组件的样式，然后是页面样式，最后是默认页面的样式。我认为正确的加载顺序应该是先加载默认页的样式，然后是子组件的样式，最后才是页面样式。有没有可能是加载顺序导致的？还是其它原因？怎样解决这个问题？

I found a strange problem，some text color style do not take effect until a few seconds after the page loaded,there's a high probability of that happening.I checked the loading order of the styles,the first thing to load is the child component's style,then there is the page style,finally the default page style.I think the correct loading order would be to load the default page style first,then there is the style of the child components,finally the page style.Is it possible that the loading order is causing this?Or is it something else?How to solve this problem?

https://user-images.githubusercontent.com/15007739/208135861-bfcc7b21-98c0-40c3-b66e-6228d08be36c.mp4

![bug](https://user-images.githubusercontent.com/15007739/208135994-a2556c75-0d94-4984-bc6d-e8959756f7cf.png)

**Comment by brillout** [comment]
It's very likely unrelated to VPS. But, if you can provide a *minimal* reproduction, then I'll have a look at it.

**Reply by kekeqy** [reply]
This may be a chrome problem,other browsers don't have this problem.
